centreon/doc/API/v23.10/Configuration/Dashboard/Schema/Dashboard.Listing.One.yaml

```diff
@@ -0,0 +1,64 @@
+type: object
+
+properties:
+
+  id:
+    type: integer
+    description: "Dashboard ID"
+    example: 1
+
+  name:
+    type: string
+    maxLength: 200
+    description: "Dashboard name"
+    example: "my dashboard"
+
+  description:
+    type: string
+    nullable: true
+    maxLength: 65535
+    description: "Description about this dashboard"
+    example: "my description"
+
+  created_by:
+    type: object
+    nullable: true
+    properties:
+      id: { type: integer, description: "Creation contact ID" }
+      name: { type: string,  description: "Creation contact name" }
+
+  updated_by:
+    type: object
+    nullable: true
+    properties:
+      id: { type: integer, description: "Last update contact ID" }
+      name: { type: string,  description: "Last update contact name" }
+
+  created_at:
+    type: string
+    format: date-time
+    nullable: true
+    description: "Creation date (ISO8601)"
+
+  updated_at:
+    type: string
+    format: date-time
+    nullable: true
+    description: "Last update date (ISO8601)"
+
+  own_role:
+    type: string
+    enum: [ viewer, editor ]
+    description: "The role current user owns on this dashboard."
+
+  global_refresh:
```

Comment:
Information redundancy : check if refresh_interval is sufficient to determine refresh_type